apps/stable_diffusion/src/models/model_wrappers.py

```diff
@@ -263,33 +261,6 @@ def forward(self, input):
         )
         return shark_vae
 
-    def get_vae_upscaler(self):
-        class VaeModel(torch.nn.Module):
-            def __init__(self, model_id=self.model_id, low_cpu_mem_usage=False):
-                super().__init__()
-                self.vae = AutoencoderKL.from_pretrained(
-                    model_id,
```

Comment:
@Shukla-Gaurav can you also take a look at this?

On closely monitoring for a better/cleaner control it seemed to me that Vae-Upscaler was basically the control flow for `use_base_vae`.

So, I've dropped this and made adjustment in `compile_models` function.